Source/logging.cpp

```diff
@@ -41,21 +41,19 @@ void __cdecl log_cleanup_mutex(void)
 	DeleteCriticalSection(&sgMemCrit);
 }
 
-void __cdecl log_flush(BOOLEAN force_close)
+void __cdecl log_flush(BOOL force_close)
 {
-	HANDLE v1;                  // eax
-	DWORD NumberOfBytesWritten; // [esp+8h] [ebp-4h]
+	DWORD NumberOfBytesWritten;
 
 	EnterCriticalSection(&sgMemCrit);
 	if (nNumberOfBytesToWrite) {
 		if (log_file == (HANDLE)-1) {
-			v1 = log_create();
-			log_file = v1;
-			if (v1 == (HANDLE)-1) {
+			log_file = log_create();
+			if (log_file == (HANDLE)-1) {

```

Comment:
WinAPI defines `#define INVALID_HANDLE_VALUE (HANDLE)-1`, could be used here :)

Reply:
:D probably haha, I have no idea that such things exist :P

Reply:
updated, thx :)